python/packages/sdk/pyproject.toml

```diff
@@ -1,5 +1,8 @@
 [build-system]
-requires = ["setuptools>=65.6.3", "wheel>=0.38.4"]
+requires = [
+    "setuptools>=65.6.3",
+    "wheel>=0.38.4",
+]
```

Comment:
Ideally if whitespace changes are not introduced, as they blur the substance of the PR

Can we cover this file with some form of _prettier_ which will ensure unified style in all cases?

Reply:
As `pyproject.toml` is a relatively new convention, it doesn't look like the standard formatting tools like `black`, `yapf`, etc support formatting package metadata, although [this project](https://pyproject-fmt.readthedocs.io/en/latest/) exists to unify style. It only gives you knobs to set the number of spaces per indent, though (default is 2 with the tool, `black` insists on 4). I can integrate it into the project easily.